pkg/tbtc/node.go

```diff
@@ -153,18 +161,70 @@ func (n *node) joinDKGIfEligible(seed *big.Int, startBlockNumber uint64) {
 				n.protocolLatch.Lock()
 				defer n.protocolLatch.Unlock()
 
-				result, _, err := n.dkgExecutor.Execute(
+				retryLoop := newDkgRetryLoop(
```

Comment:
`retryLoop` and `newDkgRetryLoop` both evoke a sense of action to me; that we'll be calling `retryLoop` with some arguments because it's *the loop itself*, and `newDkgRetryLoop` is a factory that generates loops.

Looking at what it actually represents, I think it's closer to "loop information" or "loop state", so maybe `retryLoopInfo` or `retryLoopState` something in that vein?

Reply:
I take it back!
```
func (drl *dkgRetryLoop) start(
	ctx context.Context,
	dkgAttemptFn dkgAttemptFn,
) (*dkg.Result, error) {
```
It *can* loop!